### ISSUE TITLE
Docker Image

### ISSUE THREAD

**Issue by BrianHicks** [petewarden/dstk#53]
Hi there,

I'd love to stick DSTK into our Mesos cluster, and a Docker image would be perfect for that. Has there been any work done on this? Would it be a welcome pull request?

**Comment by petewarden** [comment]
I'd love a pull request! There hasn't been any previous work that I'm aware of but I'm a big fan of Docker and Mesos, so it would be great to see, let me know how it goes.

**Comment by BrianHicks** [comment]
I'll give it a shot. I'm guessing [/docs/ec2setup.txt](https://github.com/petewarden/dstk/blob/master/docs/ec2setup.txt) would be the best place to start?

**Comment by fancyremarker** [comment]
@BrianHicks: Have you investigated this at all? I agree a Dockerfile would be quite cool, so I could take a stab at that pull request too.

**Comment by BrianHicks** [comment]
@fancyremarker no, I fell off this particular issue because of work. @petewarden would the ec2setup notes still be the best place, do you think?

**Comment by petewarden** [comment]
I think either the ec2setup or the vagrantsetup notes would be the best resources to use. They're almost identical, but maybe the Vagrant one might be slightly closer to the Docker case?

**Comment by jgujgu** [comment]
I found this. I will confirm its usability. https://github.com/korczis/docker-dstk
